external/glad/openglCfg.hpp.cmake

```
@@ -12,5 +12,8 @@
 #define RTTR_OGL_ES @RTTR_OGL_ES@
 // True(thy) if OpenGL compatibility profile should be used
 #define RTTR_OGL_COMPAT @RTTR_OGL_COMPAT@
+// True(thy) if gl4es should be used
+#define RTTR_OGL_GL4ES @RTTR_OGL_GL4ES@
```

Let's avoid the C&P of the var name now:
```suggestion
// 1 if OpenGL ES should be used
#cmakedefine01 RTTR_OGL_ES
// 1 if OpenGL compatibility profile should be used
#cmakedefine01 RTTR_OGL_COMPAT
// 1 if gl4es should be used
#cmakedefine01 RTTR_OGL_GL4ES
```

Edit: seems it doesn't show the unmodified lines in the suggestion.